src/components/footer/_macro.spec.js

```diff
@@ -478,6 +478,22 @@ describe('macro: footer', () => {
           expect($('.custom-logo').length).toBe(1);
         });
       });
+
+      it('has the additional logo image and link', () => {
+        const $ = cheerio.load(
+          renderComponent('footer', {
+            legal: EXAMPLE_LEGAL_PARAM,
+            crest: true,
+            logo: {
+              logoURL: '#0',
+              logoImage: '<img src="a-logo.svg">',
+            },
+          }),
+        );
+
+        expect($('.ons-footer__additional-poweredBy-link').attr('href')).toBe('#0');
+        expect($('.ons-footer__additional-poweredBy-link > img').attr('src')).toBe('a-logo.svg');
```

Comment:
```suggestion
        expect($('.ons-footer__poweredBy-additional-link').attr('href')).toBe('#0');
        expect($('.ons-footer__poweredBy-additional-link > img').attr('src')).toBe('a-logo.svg');
```

I think updating this class to this would fit better with the other class names now. Other than these things it looks good now